source/WindowScroller/WindowScroller.js

```diff
@@ -127,6 +131,8 @@ export default class WindowScroller extends PureComponent {
 
   // Referenced by utils/onScroll
   __handleWindowScrollEvent (event) {
+    if (!this._isMounted) return
```

Comment:
@bvaughn here's a plnkr with the problematic setup:

http://plnkr.co/edit/lquCfWiYbNnypf6JrvgW?p=preview

Just open the console and scroll fast in the preview pane (or simply drag the scrollbar fast). Let me know what you think :)